README.md

```diff
@@ -16,7 +10,17 @@ This tells you if your jobs are done. And stuff like that.
  - Gets input / output dataset names
  - Works with piping, other unix nice things
 
-## Installation
+Like badges? We have the best badges:
+
+[![GitHub Project](https://img.shields.io/badge/GitHub--blue?style=social&logo=GitHub)](https://github.com/dguest/pandamonium)
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4019463.svg)](https://doi.org/10.5281/zenodo.4019463)
+
+[![PyPI version](https://badge.fury.io/py/pandamonium.svg)](https://badge.fury.io/py/pandamonium)
+[![Supported Python versions](https://img.shields.io/pypi/pyversions/pandamonium.svg)](https://pypi.org/project/pandamonium/)
+
+
+Installation
+------------
```

Comment:
I really feel like I should encourage you to write RST more. :)

Reply:
I had to google RST, but it seems like it supports this:

https://en.wikipedia.org/wiki/ReStructuredText#Headers

I know that it doesn't give rise to rules that can work with infinite recursion, but I'm not convinced that documentation needs that anyway. My bigger concern here is just spacing things out more so it's clear even when you read it in text form.

Reply:
I suggested RST as I think you may actually like writing it in more than you do Markdown given the Markdown styles that you have adopted. RST is actually the preferred format for working with PyPI and Sphinx, so `pyhf` actually has our `README` in RST: https://raw.githubusercontent.com/scikit-hep/pyhf/master/README.rst